src/components/githubCTA/index.tsx

```diff
@@ -24,13 +24,16 @@ export function GitHubCTA() {
         welcome, whether fixing a typo (drat!) or suggesting an update ("yeah, this would
         be better").
         <div>
-          <Link href={sourceUrl}>Suggest an edit to this page</Link>{' '}
-          &nbsp;&nbsp;|&nbsp;&nbsp;
-          <Link href="https://docs.sentry.io/contributing/">Contribute to Docs</Link>{' '}
+          <Link href={sourceUrl}>Edit this page in GH</Link> &nbsp;&nbsp;|&nbsp;&nbsp;
+          <Link href="https://docs.sentry.io/contributing/">
+            Learn about contributing to docs
```

Comment:
I'm not sure how I feel about making this link text longer...

Reply:
Do you have any ideas for how to word this a different way? I think that since we're clarifying all three, it would be good to clarify this one too. I would expect "contribute to docs" to take me somewhere where I could make edits rather than learning about how to contribute... I think it's important to clarify that. And I don't think it looks too long in the preview :) but open to other ideas.

Reply:
"How to contribute" maybe?

Reply:
Great suggestion @chargome!